backend/src/scrape/agencyShortName.js

```diff
@@ -0,0 +1,216 @@
+// Shortnames are used to take up less space on the frontend, while preserving maximum information about agency Name
+// Shortname is either a Full name when shorter than or equal to X characters and Abbreviation when longer than X characters.
+// X = 16
+
+// TODO: dynamicaly create shortNames array, given length treshold.
+
+const shortNames = [
+    "BSA",
+    "CIDA-E",
+    "AEM",
+    "ASAL",
+    "AP-MCSTA",
+    "APRSAF",
+    "APSCO",
+    "ALR",
+    "AMAKA",
+    "BIRA",
+    "ABAE",
+    "AEB",
+    "UKSA",
+    "ABAE",
+    "SRI-BAS",
+    "CSA",
+    "CNSA",
+    "CCE",
+    "CRISP",
+    "CSIRO",
+    "CCSDS",
+    "COSPAR",
+    "HSA",
+    "CSO",
+    "DRC",
+    "DTU",
+    "ESA",
+    "GISTDA",
+    "DLR",
+    "HSO",
+    "ISRO",
+    "ISARS",
+    "INTA",
+    "ISA",
+    "ISA",
+    "ASI",
+    "JAXA",
+    "NSA",
+    "SRI",
+    "KCST",
+    "KARI",
+    "LSA",
+    "ANGKASA",
+    "NASA",
+    "NARSS",
+    "CNES",
+    "CONIDA",
+    "CNIE",
+    "CONAE",
+    "LAPAN",
+    "NRSC",
+    "CNT",
+    "USSRA",
+    "HKAY",
+    "NSPO",
+    "NASRDA",
+    "SRON",
+    "NRS",
+    "SUPARCO",
+    "FTC SO",
+    "ASR",
+    "CRTS",
+    "RFSA",
+    "SLSA",
+    "TUBITAK UZAY",
+    "CCCP",
+    "SPARRSP",
+    "CBK-PAN",
+    "SANSA",
+    "KACST-SRI",
+    "SSO",
+    "TNSA",
+    "UNOOSA",
+    "SSO",
+    "UNCOPUOS",
+    "SNSB",
+    "OHB System",
+    "THALES",
+    "JSC-ISS",
+    "Boeing",
+    "EADS",
+    "LMT",
+    "SSL",
+    "Amsat",
+    "ATSB",
+    "BLL",
+    "BAE",
+    "CASC",
+    "FSED",
+    "FSS",
+    "GE",
+    "HSD",
+    "HAC",
+    "IHI",
+    "IAI",
+    "KhSC",
+    "LA",
+    "MHI",
+    "NOC",
+    "OSC",
+    "PHILCO",
+    "ROI",
+    "RSC",
+    "SPAR",
+    "SpaceDev",
+    "GD",
+    "SSTL",
+    "SA",
+    "TAI",
+    "TRW",
+    "TsSKB-Progress",
+    "OKB-586",
+    "INVAP",
+    "",
+    "ASA",
+    "EADS",
+    "ELS",
+    "ILS",
+    "ISCK",
+    "SQ",
+    "SpaceX",
+    "Sea Launch",
+    "Starsem SA",
+    "ULA",
+    "PA Yuzhmash",
+    "DSI",
+    "RI",
+    "PRI",
+    "TUI",
+    "RUAG Space",
+    "Andrews Space",
+    "KDA",
+    "Aerojet",
+    "AMROC",
+    "Rocketdyne",
+    "AARC",
+    "REL",
+    "Snecma",
+    "Armadillo",
+    "Bigelow",
+    "BO",
+    "CSU",
+    "PlanetSpace",
+    "Scaled",
+    "XCOR",
+    "Canadian Arrow",
+    "RL",
+    "SSLC",
+    "IOS",
+    "Masten",
+    "SSC",
+    "UPA",
+    "MDC",
+    "NP",
+    "",
+    "ATK",
+    "BAC",
+    "Chrysler",
+    "Avio S.p.A",
+    "RAAF",
+    "USAF",
+    "PLA",
+    "VKO",
+    "",
```

Comment:
Why there are empty spaces here?

Reply:
As mentioned before, deleted agencies from the database. The array preserves the ID mappings as scraped from the database. The function should return "Unknown" on those too.